src/Components/Body.jsx

```diff
@@ -2,33 +2,35 @@ import { Route, Routes } from "react-router-dom";
 import Reviews from "./Reviews";
 import ReviewSingle from "./ReviewSingle";
 import { useState, useEffect } from "react";
-import { getReviews, getReviewID, getCategories } from "../api";
+import { getReviews, getReviewId, getCategories } from "../api";
 import { useParams } from "react-router-dom";
+import HomeBody from "./HomeBody";
+import ReviewIdComments from "./ReviewIdComments";
 
 export default function Body() {
   const [reviews, setReviews] = useState([]);
-  // const { item_id } = useParams();
-  // const [singleReview, setSingleReview] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
 
   useEffect(() => {
-    // setIsLoading(true);
+    setIsLoading(true);
     getReviews().then((data) => {
       setReviews(data);
-      // setIsLoading(false);
+      setIsLoading(false);
     });
   }, []);
 
-  // useEffect(() => {
-  //   getReviewID(item_id).then((data) => {
-  //     setSingleReview(data);
-  //   });
-  // }, [item_id]);
-
-  return (
-    <div className="Body mb-10 mt-10 p-5">
+  return isLoading ? (
+    <h1>Loading buddy</h1>
+  ) : (
+    <div className="Body mb-10 mt-10 p-5" id="body-top">
       <Routes>
+        <Route path="/" element={<HomeBody />} />
         <Route path="/reviews" element={<Reviews reviews={reviews} />} />
         <Route path="/reviews/:review_id" element={<ReviewSingle />} />
+        <Route
+          path="/reviews/:review_id/comments"
```

Comment:
not sure that comments warrants its own route; are they not better rendered with each review on path `reviews/:review_id`?

Reply:
ok, so after tinkering with the app, you're doing this anyway, so can delete comments route

Reply:
with that said, consider rendering the comments as clearly separate from the authored review, so it's clear what's review and what's user engagement